src/main/java/com/flightstats/hub/spoke/FileSpokeStore.java

```diff
@@ -246,6 +258,15 @@ private String getLatest(String channel, String limitPath, DateTime hourToSearch
         return getLatest(channel, limitPath, previous);
     }
 
+    private boolean doesNotEndWith(File file) {
```

Comment:
This method name is unclear - `isNotTempFile`?

Reply:
I might go a step further and make it `isTempFile()` and then use negation where that's called. A method with the word "Not" in it is a bit weird.

Or go the other way and name it after whatever is implied if it's not a temp file, `isStandardSpokeFile()` or some such.